app/src/main/res/values-bn/strings.xml

```diff
@@ -1,273 +1,419 @@
-    <string name="small_text">Small Text</string>
-    <string name="client_accounts">Client Accounts</string>
-    <string name="splash">Splash</string>
-    <string name="savings">Saving</string>
-    <string name="loan">Loan</string>
-    <string name="loan_account">Loan Account</string>
-    <string name="share">Share</string>
-    <string name="share_account">Share Account</string>
-    <string name="clients_list">Chose a client</string>
-    <string name="working">Working</string>
-    <string name="status_image">Status Image</string>
-    <string name="loan_repayment_schedule">Repayment Schedule</string>
-    <string name="last_transaction">Last Transaction </string>
-    <string name="made_on">Made on</string>
-    <string name="make_transfer">Make a Transfer</string>
-    <string name="select_loan_product">Select Loan Product*</string>
-    <string name="purpose_of_loan">Purpose of Loan*</string>
-    <string name="principal_amount">Principal Amount*</string>
-    <string name="amount">Amount</string>
-    <string name="remark">Remark</string>
-    <string name="expected_disbursement_date">Expected Disbursement Date</string>
-    <string name="submission_date">Submission Date</string>
-    <string name="transfer_date">Transfer Date</string>
-    <string name="apply_for_loan">Apply for Loan</string>
-    <string name="update_loan">Update Loan</string>
-    <string name="withdraw_loan">Withdraw Loan</string>
-    <string name="withdraw_loan_reason">Reason to Withdraw Loan</string>
-    <string name="loan_account_withdrawn_successfully">Loan account successfully withdrawn</string>
-    <string name="submit_loan">Submit Loan</string>
-    <string name="new_loan_application">New Loan Application for </string>
-    <string name="update_loan_application">Update Loan Application for </string>
-    <string name="loan_interest_type">Interest Type</string>
-    <string name="amortization">Amortization</string>
-    <string name="interest_calculation_period">Interest calculation period</string>
-    <string name="repayment_strategy">Repayment Strategy</string>
-    <string name="pay_to">Pay To</string>
-    <string name="pay_from">Pay From</string>
-    <string name="cancel">Cancel</string>
-    <string name="review_transfer">Review</string>
-    <string name="transfer_to">Transfer To</string>
-    <string name="transfer_from">Transfer From</string>
-    <string name="making_transfer">Making Transfer</string>
-    <string name="deposit">Deposit</string>
-    <string name="enter_amount">Enter Amount</string>
-    <string name="remark_is_mandatory">Remark is mandatory</string>
-    <string name="approved">Approved</string>
-    <string name="shares_pending">Pending</string>
-    <string name="loan_amount_paid">Paid</string>
-    <string name="balance">Balance</string>
-    <string name="rejected">Rejected</string>
-    <string name="waiting">Waiting</string>
-    <string name="overpaid">Overpaid</string>
+    <string name="login">লগ ইন করুন</string>
```

Comment:
Instead of keeping the `app_name` as Self Service App, change it to `Mifos Mobile`. Do it in all of your translation PRs

Reply:
strings.xml file in the master branch has a `<string name="app_name">Mifos Mobile</string>`, so every removed string app name from the translation files will be `Mifos Mobile`. [Here's the file in the master branch](https://github.com/letelete/mifos-mobile/blob/development/app/src/main/res/values/strings.xml)

Reply:
I've checked out all of my pr files, and all have removed app name string